site/docs/tools/sql.md

```diff
@@ -3,32 +3,26 @@ Spark SQL extensions provide an easy way to execute common Nessie commands via S
 
 ## How to use them
 
+{%- for (sparkver, iceberg_spark_runtime, nessie_spark_extensions) in [
+  ('3.3', artifacts.iceberg_spark_runtime_33_212, artifacts.nessie_spark_extensions_33_212),
+  ('3.2', artifacts.iceberg_spark_runtime_32_212, artifacts.nessie_spark_extensions_32_212),
+  ('3.1', artifacts.iceberg_spark_runtime_31_212, artifacts.nessie_spark_extensions_31_212),
+] %}
 
-### Spark 3.1
+### Spark {{sparkver}}
 
-In order to be able to use Nessie's custom Spark SQL extensions, one needs to configure
-`org.projectnessie:nessie-spark-extensions:{{ versions.java }}` along with `org.apache.iceberg:iceberg-spark-runtime-{{ versions.iceberg_spark32 }}:{{ versions.iceberg }}`.
-Here's an example of how this is done when starting the `spark-sql` shell:
-
-```
-bin/spark-sql 
-  --packages "org.apache.iceberg:iceberg-spark3-runtime-{{ versions.iceberg }}:{{ versions.iceberg }},org.projectnessie:nessie-spark-extensions:{{ versions.java }}"
-  --conf spark.sql.extensions="org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions,org.projectnessie.spark.extensions.NessieSparkSessionExtensions"
-  --conf <other settings>
-```
-
-### Spark 3.2
+In order to be able to use Nessie's custom Spark SQL extensions with Spark {{sparkver}}.x, one needs to configure
+`{{iceberg_spark_runtime.spark_jar_package}}` along with `{{nessie_spark_extensions.spark_jar_package}}`.
 
-In order to be able to use Nessie's custom Spark SQL extensions with Spark 3.2.x, one needs to configure
-`org.projectnessie:nessie-spark-3.2-extensions:{{ versions.java }}` along with `org.apache.iceberg:iceberg-spark-runtime-{{ versions.iceberg_spark32 }}:{{ versions.iceberg }}`.
 Here's an example of how this is done when starting the `spark-sql` shell:
 
-```
+``` sh
 bin/spark-sql 
-  --packages "org.apache.iceberg:iceberg-spark-runtime-{{ versions.iceberg_spark32 }}:{{ versions.iceberg }},org.projectnessie:nessie-spark-3.2-extensions:{{ versions.java }}"
-  --conf spark.sql.extensions="org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions,org.projectnessie.spark.extensions.NessieSpark32SessionExtensions"
+  --packages "{{iceberg_spark_runtime.spark_jar_package}},{{nessie_spark_extensions.spark_jar_package}}"
+  --conf spark.sql.extensions="org.apache.iceberg.spark.extensions.IcebergSparkSessionExtensions,org.projectnessie.spark.extensions.NessieSparkSessionExtensions"
   --conf <other settings>
 ```
+{%- endfor %}
```

Comment:
This is a more "sane" illustration of jinja looping.  The tabular example preceding is rough because tables are rough in markdown _(and I put a lot of formatting in there to prettify the table)_, but this one should be pretty easy on the eyes.

See what you think.